### PR TITLE
added image for activity and place

### PR DESCRIPTION
**Pull request by natiaddis**

### CODE REVIEW

File: src/main/java/com/example/tour/place/PlaceDTO.java [hunk 1]
```diff
@@ -15,4 +20,8 @@
     private double longitude;
     @JsonProperty(value= "description")
     private String description;
+    @JsonProperty(value= "images")
+    private List<Image> images;
```
Comment (ivankras) [inline src/main/java/com/example/tour/place/PlaceDTO.java:24]: Shouldn't this be private List<ImageDTO>?

**Review by ivankras — COMMENTED** [review]
Good job!
Please, check my comment!

File: src/main/java/com/example/tour/place/ImageDTO.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.Data;
+
+@Data
+public class ImageDTO {
+    @JsonProperty(value= "id")
+    private Long id;
+    @JsonProperty(value= "path")
+    private String path;
+    @JsonProperty(value= "place")
+    private PlaceDTO place;
```
Comment (ivankras) [inline src/main/java/com/example/tour/place/ImageDTO.java:13]: Should the image take a place? What about the image of an activity?

File: src/main/java/com/example/tour/place/PlaceDTO.java (hunk 1 above)
Comment (natiaddis) [reply src/main/java/com/example/tour/place/PlaceDTO.java:24]: it is a private list of images

File: src/main/java/com/example/tour/place/ImageDTO.java (hunk 2 above)
Comment (natiaddis) [reply src/main/java/com/example/tour/place/ImageDTO.java:13]: it's just to return the place object that the image belongs to if the front-end needs it

**Pull request merged**